### ISSUE TITLE
Mineralogy support

### ISSUE THREAD

**Issue by Trinsdar** [asiekierka/UnlimitedChiselWorks#35]
Would you be able to add support for mineralogy's various stones?

**Comment by mjac0by** [comment]
Until/unless proper support is ever implemented, you can at least partially address it via ModTweaker , like so:

mods.chisel.Carving.addGroup("amphibolite");
mods.chisel.Carving.addVariation("amphibolite", <mineralogy:amphibolite>);
mods.chisel.Carving.addVariation("amphibolite", <mineralogy:amphibolite_brick>);
mods.chisel.Carving.addVariation("amphibolite", <mineralogy:amphibolite_smooth>);
mods.chisel.Carving.addVariation("amphibolite", <mineralogy:amphibolite_smooth_brick>);

You don't need to .addGroup if there's already a chisel group from another mod.
This should be in a .zs file in the /scripts folder. Requires CraftTweaker to be installed also.

This does not generate new block types, but it at least makes it possible to chisel between the existing appearances, and any added from other mods.

**Comment by asiekierka** [comment]
No, the textures are very dissimilar to vanilla - and this is the trick UCW relies on for texture generation.
